solr/contrib/ltr/src/java/org/apache/solr/ltr/model/LambdaMARTModel.java

```diff
@@ -25,20 +25,78 @@
 import org.apache.lucene.search.Explanation;
 import org.apache.solr.ltr.feature.Feature;
 import org.apache.solr.ltr.norm.Normalizer;
-import org.apache.solr.ltr.util.LTRUtils;
+import org.apache.solr.util.SolrPluginUtils;
 
 public class LambdaMARTModel extends LTRScoringModel {
 
-  private final List<RegressionTree> trees = new ArrayList<RegressionTree>();
+  private final HashMap<String,Integer> fname2index;
```

Comment:
Good question, let me try and remove them (from memory they were needed I think due to how the classes are used i.e. the invokeSetters usage).